src/rime_api.h

```diff
@@ -635,6 +638,10 @@ typedef struct rime_api_t {
                                  const char* option_name,
                                  Bool state);
 
+  Bool (*hilite_candidate)(RimeSessionId session_id, size_t index);
```

Comment:
Inserting new members breaks ABI compatibility.
Don't.
Append new members to the end instead.